service/src/test/java/uk/gov/hmcts/reform/fpl/service/email/content/cmo/DraftCMOUploadedContentProviderTest.java

```diff
@@ -65,4 +65,38 @@ void shouldCreateTemplateWithExpectedParameters() {
 
         assertThat(template).usingRecursiveComparison().isEqualTo(expected);
     }
+
+    @Test
+    void shouldCreateExpectedTemplateWhenHearingJudgeIsMagistrate() {
+        List<Element<Respondent>> respondents = wrapElements(
+            Respondent.builder()
+                .party(RespondentParty.builder()
+                    .lastName("Vlad")
+                    .build())
+                .build());
+
+        JudgeAndLegalAdvisor judge = JudgeAndLegalAdvisor.builder()
+            .judgeTitle(JudgeOrMagistrateTitle.MAGISTRATES)
+            .build();
+
+        HearingBooking hearing = HearingBooking.builder()
+            .type(CASE_MANAGEMENT)
+            .startDate(LocalDateTime.of(SOME_DATE, LocalTime.of(0, 0)))
+            .judgeAndLegalAdvisor(judge)
+            .build();
+
+        DraftCMOUploadedTemplate template = contentProvider.buildTemplate(
+            hearing, CASE_NUMBER, judge, respondents, "123456"
+        );
+
+        DraftCMOUploadedTemplate expected = DraftCMOUploadedTemplate.builder()
+            .judgeName("")
+            .judgeTitle("Justice of the Peace")
+            .respondentLastName("Vlad")
+            .subjectLineWithHearingDate("Vlad, 123456, case management hearing, 20 February 2020")
+            .caseUrl(caseUrl(CASE_NUMBER.toString(), DRAFT_ORDERS))
+            .build();
+
+        assertThat(template).usingRecursiveComparison().isEqualTo(expected);
```

Comment:
don't need to use recursive comparison, the data annotation provides an implementation for equals and hashcode so that when .equals is called it will compare fields instead of reference